cluster/userdata-master.yaml

```diff
@@ -173,6 +172,9 @@ coreos:
         [Install]
         WantedBy=multi-user.target
 
+    - name: locksmithd.service
+      mask: true
```

Comment:
The `update-operator` takes over the role of `locksmithd`. `update-engine` which polls `coreos.com` for updates is running.